scripts/prepare_compliance_data.py

```diff
@@ -0,0 +1,155 @@
+"""
+A script that takes the compliance dataset (the released version
+of which is at https://github.com/ga4gh/compliance/tree/master/test-data)
```

Comment:
@dcolligan : It's not downloaded from there because one of its principal uses is by GA4GH developers to develop NEW compliance tests with new data: Thus, as they work on a dataset/tests, they can point the script at their local working directory version.